### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Leela0o5**

### CODE REVIEW

**Comment by Leela0o5** [comment]
#3

File: AlphabetClassification/app.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+# Input Widges which only consist of Top features as input.
+input_features = pd.DataFrame([{feature:st.sidebar.slider(
+    feature, float(df[feature].min()),float(df[feature].max()), float(df[feature].mean())) 
+    for feature in top_features_names}],columns=top_features_names)
+st.subheader('Input Features')
+st.write(input_features)
+
+
+# Prediction part
+number_to_alphabet = {i: chr(65+i) for i in range(26)}
```
Comment (SaranDharshanSP) [inline AlphabetClassification/app.py:84]: Actually the classification is  between "A" and "B" only

**Review by SaranDharshanSP — CHANGES_REQUESTED** [review]
The streamlit looks great. its just that the classifcation here is only between 2 classes so do the necessary changes.

**Comment by SaranDharshanSP** [comment]
And make it into one PR i see another one there ...

File: AlphabetClassification/app.py (hunk 1 above)
Comment (Leela0o5) [reply AlphabetClassification/app.py:84]: Hey @SaranDharshanSP, I'll make the required changes. But I'm wondering if we should focus only on classifying "A" and "B" or include "C" as well. Also, after making the changes, should I create a new demo for this specific classification, or should we keep the previous one that classifies all the alphabets?

Comment (SaranDharshanSP) [reply AlphabetClassification/app.py:84]: > Hey @SaranDharshanSP, I'll make the required changes. But I'm wondering if we should focus only on classifying "A" and "B" or include "C" as well. Also, after making the changes, should I create a new demo for this specific classification, or should we keep the previous one that classifies all the alphabets?

A and B only as other issues too are trying on A/B only.

**Pull request closed without merging**